client/router.js

```diff
@@ -36,12 +37,20 @@ Router.prototype.route = function(path, options) {
   };
 
   this._routes.push(route);
+  if (options.name) {
+    this._pathsMap[options.name] = path;
+  }
+
   this._updateCallbacks();
 
   return route;
 };
 
 Router.prototype.path = function(pathDef, fields, queryParams) {
+  if (this._pathsMap[pathDef]) {
```

Comment:
We don't care subscription name clash. And we don't care wrong subscription name on check readiness (just return false). So I think it's better to follow one convention.

If we care clash name and a path, we should also worry about clash between name with other names etc ...

What do you think?